tensorflow/python/data/ops/dataset_ops.py

```diff
@@ -648,6 +648,12 @@ def concatenate(self, dataset):
   def prefetch(self, buffer_size):
     """Creates a `Dataset` that prefetches elements from this dataset.
 
+    Note: Like other `Dataset` methods, prefetch operates on the elements of
+    the dataset. It has no concept of examples vs. batches. `examples.prefetch(2)`
```

Comment:
This and the next line violate the 80 character limit.

Reply:
thx, done